recipe/recipe_13TeV.sh

```diff
@@ -26,5 +26,27 @@ if [ "$HZZ" = "1" ]; then
     git clone -b 74x-root6 https://github.com/cms-analysis/HiggsAnalysis-CombinedLimit.git HiggsAnalysis/CombinedLimit
 fi
 
+# svFit packaged checked out for everyone so that svFit code in FSA compiles
+git clone git@github.com:veelken/SVFit_standalone.git TauAnalysis/SVfitStandalone
+pushd TauAnalysis/SVfitStandalone
+git checkout svFit_2015Apr03
+popd
+
+if [ "$HTT" = "1" ]; then
```

Comment:
This doesnt change anything in the code right? If not, dont hide it behind the HTT.
(By change anything, I mean, does it change the standard met variables, or does it just add the ability to do MET PU subtraction?)